controllers/monitoring_alert.go

```diff
@@ -180,7 +213,15 @@ func (r *ManagedMCGReconciler) reconcileMonitoringResources() error {
 	}
 	for i := range promRuleList.Items {
 		obj := promRuleList.Items[i]
-		utils.AddLabel(obj, monLabelKey, monLabelValue)
+		if obj.GetName() == "noobaa-prometheus-rules" {
+			r.Log.Info("found noobaa prometheus rules, checking for inclusion")
+			if v, ok := obj.GetLabels()[monLabelKey]; ok && v == monLabelValue {
+				utils.RemoveLabel(obj, monLabelKey)
+				r.Log.Info("removing app label from noobaa prometheus rules")
```

Comment:
IMHO, a single quote will make more easy to understand the message. ---> "removing 'app' label", instead "removing app label"